x-pack/plugins/search_assistant/public/types.ts

```diff
@@ -15,6 +19,10 @@ export interface SearchAssistantPluginSetup {}
 export interface SearchAssistantPluginStart {}
 
 export interface SearchAssistantPluginStartDependencies {
+  licensing: LicensingPluginStart;
+  ml: MlPluginStart;
```

Comment:
should `ml` be listed as a required plugin in `kibana.jsonc`, not sure if it's always there.

Reply:
Done in https://github.com/elastic/kibana/pull/196832/commits/c82346937531f113f1098113936f38aacef19dde